### ISSUE TITLE
Cannot reload the model after using mobilenetV2 in Tensorflow Hub

### ISSUE THREAD

**Issue by zlwywmzyq** [tensorflow/hub#309]
I apply a transfer learning from https://tfhub.dev/google/imagenet/mobilenet_v2_100_224/classification/2.
After model compiling and fitting, I saved the model as 'my_model.h5', however, when I load the file and evaluate the new model, there is FailedPreconditionError in the picture.
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/37170603/58546917-bd337380-8249-11e9-9c7d-6f3b3f438692.png">
My code are listed below.

feature_extractor_url = "https://tfhub.dev/google/imagenet/mobilenet_v2_100_224/classification/2" #@param {type:"string"}
def feature_extractor(x):
    feature_extractor_module = hub.Module(feature_extractor_url)
    return feature_extractor_module(x)
IMAGE_SIZE = (hub.get_expected_image_size(hub.Module(feature_extractor_url)) )
features_extractor_layer = keras.layers.Lambda(module, input_shape=IMAGE_SIZE+[3])
features_extractor_layer.trainable = False
init = tf.global_variables_initializer()
import tensorflow.keras.backend as K
model = tf.keras.Sequential()
model.add(features_extractor_layer)
model.add(keras.layers.Dense(640, activation='relu'))
model.add(keras.layers.Dense(640, activation='relu'))
model.add(keras.layers.Dense(20, activation='sigmoid'))
sess.run(tf.global_variables_initializer())
model.compile(
  optimizer=tf.keras.optimizers.Adam(), 
  loss='binary_crossentropy',
  metrics=['categorical_accuracy'])
sess = K.get_session()
model.fit(train_image_label_ds, epochs=1,validation_data=validation_image_label_ds)
model.save('my_model.h5')
new1_model = tf.keras.models.load_model('my_model.h5')
new1_model.evaluate(test_image_label_ds)

My computer is Mac OS with 1.14.1-dev20190520 tensorflow version。

By the way, if I initialize the model with normal Dense layers, the model.save() function works fine.

**Comment by gowtham-kp** [comment]
@zlwywmzyq Looks like the variables have not been initialized. Have you tried adding sess.run(tf.global_variables_initializer()). Can you please take a look at the similar [issue](https://github.com/tensorflow/tensorflow/issues/20803) and let me know if it helps. Thanks!

**Comment by gowtham-kp** [comment]
Closing this issue as this has been is "awaiting response" for more than 3 days. Please add additional comments and we can reopen this issue again. Thanks!
